docs/convenience_commands.md

```diff
@@ -126,26 +163,107 @@ phg merge-hvcfs \
-| `--conda-env-prefix` | Prefix for the Conda environment to use. If provided, this should be the full path to the Conda environment.          | _Current active Conda environment_         |
+| Parameter name       | Description                                                                                                         | Default value                      | Required?        |
+|----------------------|---------------------------------------------------------------------------------------------------------------------|------------------------------------|------------------|
+| `--db-path`          | Folder name where TileDB datasets and AGC record is stored. If not provided, the current working directory is used. | `""`                               | :material-check: |
+| `--output-file`      | Path and/or filename for the samples list file.                                                                     | `""`                               | :material-check: |
+| `--data-set`         | Storage from which to pull sample names. Must be one of `all`, `agc`, `gvcf`, `hvcf`                                | `hvcf`                             |                  |
+| `--conda-env-prefix` | Prefix for the Conda environment to use. If provided, this should be the full path to the Conda environment.        | _Current active Conda environment_ |
+
+**Example output**
+
+```
+B73
+Ki11
+Mo18W
+Ki3
+```
+
+
+### Create a table of haplotype IDs by reference range
+
+> Creates a tab-delimited table of haplotype IDs by reference range
+> coordinats and sample IDs
+
+**Command** - `sample-hapid-by-range`
+
+**Example**
+
+```shell
+phg sample-hapid-by-range \
+    --input-dir my/hvcf/directory \
+    --output-file output/hapid_table
+```
+
+**Parameters**
+
+| Parameter name | Description                                  | Default value | Required?        |
+|----------------|----------------------------------------------|---------------|------------------|
+| `--input-dir`  | Path to directory holding hVCF files.        | `""`          | :material-check: |
+| `--output-dir` | Path and/or filename for haplotype ID table. | `""`          | :material-check: |
+
+**Example output**
+
+```
+#CHROM START   END   B73                                SEEDGWAS1                          SEEDGWAS10
+chr1   1       5000  <c9ecfe3967a71282f3ad7c41d48e0bbf> <b19364bc9a4c07a80986b1ee181446c2> <5c8e72b2e9f11ecc652d5b8e8d0e5bf3>
+chr1   5001    6000  <f162e742c4d30f151ae6276fbebe762c> <fdfdaa361c39cf5b6f13fad195d0e519> <283a8261c193212fd5cf43d208673322>
+chr1   6001    9000  <471d4abbf0545dede647e65915345648> <d6dd5ecea7fb4e6f77f9e630f601b7a8> <13e0ac1a8d12e1aedd6a5302d1e221fd>
+```
+
+
+
+### Create a table of haplotype IDs to sample
+
+> Creates a tab-delimited table of haplotype IDs to sample gamete. Can
+> be one or multiple samples mapping to each haplotype ID.
+
```

Comment:
Should make it clear that this is intended for use with hvcf files created from aligning assemblies.  While it will work with hvcf files from imputation, ALL the sample names will be the imputed file name and not the sample names associated with the hapids when they were created.

Reply:
Added note describing this